src/libs/actions/IOU.js

```diff
@@ -235,11 +318,11 @@ function requestMoney(report, amount, currency, payeeEmail, participant, comment
             amount,
             currency,
             comment: parsedComment,
-            iouReportID: moneyRequestReport.reportID,
+            iouReportID: iouReport.reportID,
             chatReportID: chatReport.reportID,
             transactionID: optimisticTransaction.transactionID,
-            reportActionID: optimisticReportAction.reportActionID,
-            createdReportActionID: isNewChat ? optimisticCreatedAction.reportActionID : 0,
+            reportActionID: optimisticIOUAction.reportActionID,
+            createdReportActionID: isNewChatReport ? optimisticCreatedActionForChat.reportActionID : 0,
```

Comment:
@luacmartins I think we need to pass the created report action for the IOU report too. I am getting two created report actions here 
<img width="1078" alt="image" src="https://github.com/Expensify/App/assets/36083550/f2640dff-8fda-4137-9693-c6905ecaff95">

<img width="1047" alt="image" src="https://github.com/Expensify/App/assets/36083550/27a0024c-8a89-4f90-b742-4cb25ad960ae">


We wont be rendering the Created report actions so this problem technically wont eb visible so is that the intention?

Reply:
Ideally will be handled here https://github.com/Expensify/App/pull/18775#pullrequestreview-1425452759

Reply:
Updated to pass the param. I'll update Auth and Web-E too